dotCMS/src/main/java/com/dotmarketing/portlets/contentlet/business/Contentlet.java

```diff
@@ -46,7 +46,7 @@
 
     private FieldAPI fAPI = APILocator.getFieldAPI();
 
-    private String contentTypeId;
+    private String structureInode;

```

Comment:
Why include these changes?  I would rather have it called contentTypeId in our pojo

Reply:
in a previous commit the structureInode was rename to contentTypeId, them we realized it is breaking the starter (importer), so doing the fix